system-metrics/handler.go

```diff
@@ -0,0 +1,125 @@
+package function
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+
+	"github.com/openfaas/faas/gateway/metrics"
+)
+
+type Metrics struct {
+	Success int `json:"success"`
+	Failure int `json:"failure"`
+}
+
+// Handle a serverless request
+func Handle(req []byte) string {
+	fnName, err := parseFunctionName()
+
+	if err != nil {
+		log.Fatalf("couldn't parse function name from query: %t", err)
+	}
+
+	host := os.Getenv("prometheus_host")
```

Comment:
I'll add this now, but we really need a solution to #186 so that every URL is built using that suffix. Otherwise there will be so many places to do edits depending on Swarm/K8s or a non-standard namespace. cc @stefanprodan